lib/psm.c

```diff
@@ -104,7 +104,7 @@ static rpmRC markReplacedFiles(const rpmpsm psm)
 
 	if (!headerGet(h, RPMTAG_FILESTATES, &secStates, HEADERGET_MINMEM))
 	    continue;
-	
+
```

Comment:
Ignore the editor autoformat noise.

Reply:
For submitting patches, you need to get rid of any editor caused artifacts. Patches with random whitespace changes will not get merged.